.github/workflows/scripts/check-collector-module-version.sh

```diff
@@ -0,0 +1,53 @@
+get_collector_version() {
```

Comment:
Can you add a shebang?
```suggestion
#! /bin/bash

get_collector_version() {
```

Also, should we `set -euxo pipefail`?

Reply:
Sorry, forgot to round off the file before pushing. Done.